addons/mail/static/src/js/wysiwyg.js

```diff
@@ -0,0 +1,97 @@
+/** @odoo-module **/
+
+import { renderToElement } from "@web/core/utils/render";
+import { Wysiwyg } from "@web_editor/js/wysiwyg/wysiwyg";
+import { closestBlock, setCursorEnd } from "@web_editor/js/editor/odoo-editor/src/OdooEditor";
+import { patch } from "@web/core/utils/patch";
+import { MentionList } from "@mail/core/web/mention_list";
+import { url } from "@web/core/utils/urls";
+import { usePopover } from "@web/core/popover/popover_hook";
+
+patch(Wysiwyg.prototype, {
+    setup() {
+        super.setup();
+        if (this.props.options.recordInfo?.res_model === "mail.compose.message") {
+            this.mentionList = usePopover(MentionList, {
+                position: `bottom-start`,
+                onClose: () => {
+                    this.focus();
+                },
+            });
+        }
+    },
```

Comment:
Ok with current specs. Note that using a component hook conditionally on props is in theory weak, because this doesn't take OWL reactivity into account. This is actually a big drawbacks of component hooks compared to sub-components: reactivity should be managed internally of the hook